app/src/main/java/com/swent/assos/ui/components/NewsItem.kt

```diff
@@ -38,10 +38,8 @@ fun NewsItem(news: News, navigationActions: NavigationActions) {
           Modifier.semantics { testTagsAsResourceId = true }
               .testTag("NewsItem")
               .padding(0.dp)
-              .border(
-                  width = 0.5.dp,
-                  color = MaterialTheme.colorScheme.surfaceVariant,
-                  shape = RoundedCornerShape(12.dp))) {
+              .shadow(elevation = 6.dp, shape = RoundedCornerShape(12.dp))
+              .shadow(elevation = 2.dp, shape = RoundedCornerShape(12.dp))) {
```

Comment:
Do you really need two shadows ? And can you create a constant to avoid magic number if these shadows are used everywhere ?